src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MuTect2.java

```diff
@@ -954,23 +908,6 @@ private Haplotype createReferenceHaplotype(final AssemblyRegion assemblyRegion,
         return ReferenceConfidenceModel.createReferenceHaplotype(assemblyRegion, assemblyRegion.getAssemblyRegionReference(referenceReader), paddedReferenceLoc);
     }
 
-    /**
-     * Clean up reads/bases that overlap within read pairs
-     *
-     * @param reads the list of reads to consider
-     */
-    private void cleanOverlappingReadPairs(final List<GATKRead> reads, final Set<String> normalSampleNames) {
```

Comment:
Since HC uses it inside `finalizeRegion`, the natural place to deal with this would be when we fix the code duplication between that and M2's equivalent method, issue #670.